templates/footer.html

```diff
@@ -0,0 +1,5 @@
+				</div>
```

Comment:
I see...
So we just keep the <head> in header?
I'm trying to separate entirely the whole theme from the actual logic/data.